src/Models/Abstract/AbstractModel.ts

```diff
@@ -17,12 +18,39 @@ export abstract class AbstractModel extends TreeNode<AbstractModel> {
         this.type = json.type || this.type;
 
         this.schemaCheckResult = this.shallowCheckSchema(json);
+        // this.schemaCheckResult = this.deepCheckSchema();
 
         if (this.context) {
             this.context.fit = 'content';
         }
 
         this.outputSchemaMessage();
+
+        if (parent === undefined) {
+
+            if (this.type === undefined) {
+                this.path = 'undefined type >>';
+            } else {
+                this.path = this.type + ' >> ';
+            }
+
+        } else {
+            if (this.type === undefined) {
+                this.path = parent.path + 'undefined type >>';
```

Comment:
I'd prefer to use an array to describe the path, e.g. ["Container", "ColumnSet", "Column", "undefined"]. And format to "Container >> ColumnSet >> Column >> undefined" on outputing